src/App.js

```diff
@@ -1,23 +1,133 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+
+import Header from "./components/Header";
+import DisplayAlgo from "./components/DisplayAlgo";
+import DisplayAlgoLine from "./components/DisplayAlgoLine";
+import Footer from "./components/Footer";
+import { useState, useEffect } from "react";
```

Comment:
General pattern, you should have imports to external libraries before the local/relative imports:

```suggestion
import { useState, useEffect } from "react";

import Header from "./components/Header";
import DisplayAlgo from "./components/DisplayAlgo";
import DisplayAlgoLine from "./components/DisplayAlgoLine";
import Footer from "./components/Footer";

import "./App.css";
```